### PR TITLE
Migrating dropdown to ucpadmissions

### PR DESCRIPTION
**Pull request by saboreo**
Migrated dropdown from tbd storybook to ucpadmissions.

### CODE REVIEW

**Comment by lou429** [comment]
All of the code looks fine, could add more props to improve styling of the component.

File: src/components/SelectBox.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+
+class SelectBox extends Component {
+
+
+
```
Comment (raegar) [inline src/components/SelectBox.js]: Lots of space here. Don't forget to run Format Document occasionally.

File: src/stories/SelectBox.stories.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import SelectBox from '../components/SelectBox';
+
+export const box2 = () => <SelectBox list={["2", "3", "4"]}/>
```
Comment (raegar) [inline src/stories/SelectBox.stories.js]: box2 is not very descriptive of what the "Story" is. What is this story demonstrating?

**Comment by raegar** [comment]
Thanks for making changes. Looks good to merge now

**Pull request merged**